controlplane/sampler/sampler.go

```diff
@@ -12,6 +12,33 @@ import (
 	"github.com/neblic/platform/logging"
 )
 
+var (
+	capabilities = control.Capabilities{
```

Comment:
The capabilities is something that the sampler provides to the controlplane since it is the one that actually implements them. I think the definition of what capabilities the sampler support should live together with their implementation and be imported here.

They could be part of the options and have the sampler package provide them when it initializes the sampler controlplane. Otherwise, as they are now, they are hardcoded in here for all implementations.

Reply:
Changes performed as part of 862f1abc2f2364c447392570d074d33d97a0a64a